README.md

```diff
@@ -4,103 +4,99 @@ This package provides a set of language helpers, utility functions, and classes
+across platforms. `dojo-core/has` provides a simple feature detection API that makes it easy to detect which platforms
+support which features.
 
 ### Language Utilities
 
 The core package provides several modules offering a number of langauge utilities.  Some of these are heavily based
 on methods in the ES2015 proposal; others are additional APIs for commonly-performed tasks.
 
-#### array
-
-The `array` module contains analogues to the following ES2015 Array APIs:
+#### [array](docs/array.md)
 
-* `copyWithin` - Copies a sequence of elements to another position in the given array
-* `from` - Creates an `Array` from an array-like object
-* `fill` - Fills some or all elements of an array with a given value
-* `find` - Returns the first value in the array satisfying a given function
-* `findIndex` - Returns the first index in the array whose value satisfies a given function
-* `of` - Creates an `Array` with the given arguments as its elements
+The `dojo-core/array` module contains analogues to some of the ES2015 Array APIs.
 
-#### lang
+#### [lang](docs/lang.md)
 
-The `lang` module contains various utility functions for tasks such as copying objects and creating late-bound
-or partial applications of functions.
+The `dojo-core/lang` module contains various utility functions for tasks such as copying objects and creating late-bound
+or partially applied functions.
 
-#### math
+#### [math](docs/math.md)
 
-The `math` module contains analogues to a number of ES2015 APIs, including many trigonometric and logarithmic
+The `dojo-core/math` module contains analogues to a number of ES2015 APIs, including many trigonometric and logarithmic
 functions.
 
-#### string
+#### [string](docs/string.md)
 
-The `string` module contains analogues to the following ES2015 String APIs:
+The `dojo-core/string` module contains analogues to the some of the ES2015 String APIs.
 
-* `codePointAt` - Returns the UTF-16 encoded code point value of a position in a string
-* `endsWith` - Determines whether a string ends with the given substring
-* `fromCodePoint` - Creates a string using the specified sequence of code points
-* `includes` - Determines whether a string includes the given substring
-* `repeat` - Returns a string containing a string repeated a given number of times
-* `startsWith` - Determines whether a string begins with the given substring
+#### [UrlSearchParams](docs/UrlSearchParams.md)
 
-Special thanks to Mathias Bynens for granting permission to adopt code from his
-[`codePointAt`](https://github.com/mathiasbynens/String.prototype.codePointAt),
-[`fromCodePoint`](https://github.com/mathiasbynens/String.fromCodePoint), and
-[`repeat`](https://github.com/mathiasbynens/String.prototype.repeat) polyfills.
+The `dojo-core/UrlSearchParams` class can be used to parse and generate URL query strings.
 
-The `string` module also contains the following utility functions:
+#### [Event handling](docs/events.md)
```

Comment:
If this is just going to link to a doc about `on`, call the doc `on.md` and make the heading `on` (just like lang, math, string, etc. above).